boards/STM32F407ZET6-STM32_F4XX.md

```diff
@@ -0,0 +1,452 @@
+---
+title: "STM32F407ZET6 - STM32_F4XX board"
+---
+
+# {{ page.title }}
+
+![STM32_F4XX board]({{ site.url }}/assets/img/boards/STM32F407ZET6-STM32_F4xx_1.jpg)
+
+## Overview
+
+<table>
+    <tr>
+        <td rowspan="3"><b>Board</b></td>
+        <td>Name</td>
+        <td>STM32_F4XX</td>
+    </tr>
+    <tr>
+        <td>Brand</td>
+        <td>Unknown</td>
+    </tr>
+    <tr>
+        <td>Origin</td>
+        <td>China</td>
+    </tr>
+    <tr>
+        <td rowspan="6"><b>Microcontroller</b></td>
+        <td>Part</td>
+        <td>STM32F407ZET6</td>
+    </tr>
+    <tr>
+        <td>Package</td>
+        <td>LQFP144</td>
+    </tr>
+    <tr>
+        <td>Core</td>
+        <td>Cortex-M4</td>
+    </tr>
+    <tr>
+        <td>FLASH</td>
+        <td>512KB (There is a STM32F4ZGT6 variant with 1024KB)</td>
```

Comment:
This also impacts the size of available SRAM.

Reply:
Actually no, according to table 2 on page 14 in the datasheet all versions og the F405 and F407 have 112+16KB SRAM + 64KB CCM SRAM. I don't know why they split it 112 + 16 as these two segments are contiguos and (both) aliased by bit-banding. (Figure 18 on page 71 in the datasheet). Oh, found it: The 112KB SRAM can be remapped to also appear at adr 0x0000 0000 when booting from SRAM (BOOT0=1, BOOT1=1), but the 16KB are 'left behind'.

Reply:
Oh, I see.

Also, didn't having two banks of SRAM solve some problems with DMA? So that the DMA can read/write to SRAM without blocking the CPU?